config/core/test-configs.yaml

```diff
@@ -896,7 +896,7 @@ device_types:
     # no console in tinyconfig
       - blocklist:
           defconfig: ['tinyconfig']
-          kernel: ['v4.', 'v5.1', 'v5.2']
+          kernel: ['v4.', 'v5.2']
```

Comment:
I think you can just drop the `v5.1` or `v5.2` as the LTS versions are v4.9 which is already blocked, and then v5.4.  So there just won't be kernels between v5.1 and v5.4 tested on kernelci.org.

Also if the filter had to be really for blocking anything up to v5.1 it could be changed to a `regex` instead of a `blocklist`.